server/src/test/java/org/elasticsearch/search/aggregations/bucket/geogrid/InternalGeoHashGridTests.java

```diff
@@ -56,7 +56,7 @@ protected InternalGeoHashGrid createTestInstance(String name,
             double longitude = randomDoubleBetween(-180.0, 180.0, false);
 
             long geoHashAsLong = GeoHashUtils.longEncode(longitude, latitude, 4);
-            buckets.add(new InternalGeoHashGrid.Bucket(geoHashAsLong, randomInt(IndexWriter.MAX_DOCS), aggregations));
+            buckets.add(new InternalGeoHashGrid.Bucket(GeoHashType.GEOHASH, geoHashAsLong, randomInt(IndexWriter.MAX_DOCS), aggregations));
```

Comment:
If we randomize GeoHashType here at some point and check the bucket key (as GeoPoint) after the xContent roundtrip in ParsedGeoHashGrip.Bucket, I think the xContent tests will currently fail.